### ISSUE TITLE
License information is not loaded from appsettings.json after renewal

### ISSUE THREAD

**Issue by dalager** [Jumoo/uSync.Complete.Issues#233]
**Describe the bug**

After renewing our agency license (it had expired) and redeploying the site with the correct license info in `appsettings.json` the license was still invalid.

I needed to manually type in the license and hit save in the usync/dashboard/license settings page in order for the license to go "valid".

Configured as per the [docs](https://docs.jumoo.co.uk/usync/complete/setup/licence) 
```json
"uSync": {
    "Publisher": {
      "my":"othersettings"
    },
    "Complete": {
      "License": {
        "Key": "MyKeyStandaloneOrRenewing",
        "Domains": "MyDomain"
      }
    }
  }
```

**Version :**
 - Umbraco Version: 13.3.0
 - uSync Version: 13.2.1
 - uSync.Complete Version: 13.1.3

**To Reproduce**
- my licenses are fine now, cannot reproduce

**Comment by KevinJump** [comment]
hi, thanks, I *think* i know what is going on and will take a look 👍

**Comment by KevinJump** [comment]
Update : Sorry missed this - it was fixed for 13.1.5
